### PR TITLE
FixStartStateBounds: Copy attached bodies when adapting the start state

### PR DESCRIPTION
**Pull request by rhaschke**
This fixes https://github.com/ros-planning/moveit_task_constructor/issues/211, where an attached object was "lost" during planning. Turned out that the `FixStartStateBounds` adapter, while creating a new _non-diff_ `start_state`, did not copy all the attached objects, which are part of the RobotState as well. Thus with new joints, also attached objects were overridden.
Interestingly this bug was undetected over all the years...

### CODE REVIEW

**Comment by rhaschke** [comment]
I guess this will fix https://github.com/ros-planning/moveit/issues/2262 as well, because the object was attached there too.

**Comment by felixvd** [comment]
Wow. Could you summarize how to reproduce this problem? What kind of test would have caught this?

**Comment by rhaschke** [comment]
Seems to be a rare corner case. You need to have:
- a start state that is slightly outside the joint limits
- an object attached to the robot
- the attached object (and only that one) causing a collision that will be slip through unnoticed during planning
- FixStartStateBounds adapter enabled

Interestingly, the rviz display seems to correctly display the motion - including the attached object going through an obstacle. I have seen corresponding videos in the past, e.g. in #2262.
However, planning actually didn't observe a collision, because the attached object wasn't there during planning.

**Comment by felixvd** [comment]
I see. If you have that case prepared somewhere, how about making it into a test? :)

**Comment by rhaschke** [comment]
I don't have a test case lying around for this. As written in the initial comment, the issue arose from https://github.com/ros-planning/moveit_task_constructor/issues/211.

**Review by JafarAbdi — APPROVED** [review]

**Review by AndyZe — APPROVED** [review]
